kotest-framework/kotest-framework-multiplatform-plugin-gradle/src/main/kotlin/io/kotest/framework/multiplatform/gradle/kotestPluginConstants.kt

```diff
@@ -0,0 +1,5 @@
+// Do not edit manually. This file was created by :kotest-framework:kotest-framework-multiplatform-plugin-gradle:updateKotestPluginConstants
```

Comment:
This file shouldn't be checked in - instead, we should let Gradle generate it when it's needed.